pytest_plugins/manual_skipped.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+import pytest
+
+from robottelo.constants import NOT_IMPLEMENTED
+
+LOGGER = logging.getLogger(__name__)
+
+
+def pytest_collection_modifyitems(session, config, items):
+    """
+    Collects tests based on pytest option to select tests marked as failed on Report Portal
+    """
+    manualskip_args = config.getvalue('mark_manuals_skipped')
+    if not manualskip_args:
+        return
+    for item in items:
+        if item.get_closest_marker(name='stubbed'):
+            item.add_marker(marker=pytest.mark.skip(reason=NOT_IMPLEMENTED))
+
+
+def pytest_addoption(parser):
+    """Add options to pytest to modify manual tests as skipped"""
+    help_script = 'Mark stubbed tests as skipped tests.'
+    parser.addoption("--mark-manuals-skipped", nargs='?', const=True, help=help_script)
```

Comment:
Why use `nargs` and `const` here, instead of just `store_true` to create a simple flag?